recipes/cppcheck/all/test_package/conanfile.py

```diff
@@ -14,8 +12,8 @@ def build_requirements(self):
     def test(self):
         if can_run(self):
             self.run("cppcheck --enable=warning,style,performance --std=c++11 .",
-                     cwd=self.source_folder, run_environment=True)
+                     cwd=self.source_folder)
             if self.settings.os == "Windows":
-                self.run(f"{sys.executable} %CPPCHECK_HTMLREPORT% -h", run_environment=True)
+                self.run("set CPPCHECK_HTMLREPORT")
             else:
-                self.run("cppcheck-htmlreport -h", run_environment=True)
+                self.run("which cppcheck-htmlreport")
```

Comment:
Why do you changed `cppcheck-htmlreport -h` to `which cppcheck-htmlreport` ?

Reply:
cppcheck-htmlreport is a python script, where the 3rd party dependency pygments is missing. In conan v2 pipeline environment, that package is missing and and import error occures

Reply:
@jcar87 @uilianries is it expected that imports changed in the b2 pipeline?